api/e2e/test/routers_suite_test.go

```diff
@@ -11,10 +12,15 @@ import (
 	"testing"
 	"text/template"
 
+	routerConfig "github.com/caraml-dev/turing/engines/router/missionctl/config"
```

Comment:
nit: Turing related imports should be in the following block of imports

Reply:
updated